### DISCUSSION TITLE
New Mopup release soon?

### DISCUSSION THREAD

**Discussion by indyjason79** [LuckyDucko/Mopups#138] [General]
I've seen there were a couple bugs fixed recently and merged into master. When can we expect a new release with these fixes? We are going to beta next week and going live next month and I'd love to get these in before then :D

**Comment by IeuanWalker** [comment]
@indyjason79 in the mean time you can just clone the repo, and reference the project directly rather than using the NuGet package

**Comment by Thibault-Vellicus** [comment]
Do we have any fix for this ticket in the new release ? https://github.com/LuckyDucko/Mopups/issues/82
This bug makes Mopups unusable in my project :-(
